packages/fast-components-react-msft/src/carousel/carousel.tsx

```diff
@@ -58,6 +63,11 @@ class Carousel extends Foundation<
         items: void 0,
     };
 
+    /**
+     * Initial slide transition direction is none (on carousel load)
+     */
+    private slideTransitionDirection: SlideTransitionDirection | "" = "";
```

Comment:
Should this be null? Empty string seems odd.

Reply:
Sure, I'll switch to null